### DISCUSSION TITLE
Petio not working after reboot

### DISCUSSION THREAD

**Discussion by phil9309** [tteck/Proxmox#3237] [Help Needed]
Hi, I installed Petio LXC with the default settings.

Whenever I reboot my server, the Petio LXC is starting correctly, but I cant reach the GUI, the console however works.

I have to run the update script from the petio console to fix it.

Anyone else with this problem?

**Comment by info-path** [comment]
Same problem, after restarting the container Petio does not respond.

**Reply by tteck** [reply]
IP changed?

**Reply by phil9309** [reply]
Nope, IP stays the same, it's just not reachable until I run "update".

**Reply by tteck** [reply]
Made changes that _may_ solve your issue. https://github.com/tteck/Proxmox/commit/9c475a02088e436112d4458218d508157011811a

Copy and paste within the LXC console, reboot.
```
cat <<EOF >/etc/systemd/system/petio.service
[Unit]
Description=Petio a content request system
After=network.target mongod.service

[Service]
Type=simple
User=petio
Restart=on-failure
RestartSec=1
ExecStart=/opt/Petio/bin/petio-linux

[Install]
WantedBy=multi-user.target
EOF
```

**Reply by phil9309** [reply]
> Made changes that _may_ solve your issue. 

Sadly didn't change anything. I had to use "update" again, it's always downloading the complete lastest zip and replacing everything outside of the database it seems.

Let me know if or how I can provide you any more details.

**Reply by phil9309** [reply]
Oh, one more observation:

When updating Petio with the other LXCs in bulk via your script (https://tteck.github.io/Proxmox/#proxmox-ve-lxc-updater), it tells me that no new updates are available, the "update" function in the LCX itself always seems to trigger a complete reinstall. Something is a little off here, compared to the other scripts.
